xbmc/filesystem/FileCache.cpp

```diff
@@ -166,10 +166,8 @@ bool CFileCache::Open(const CURL& url)
 
   CLog::Log(LOGDEBUG,"CFileCache::Open - opening <%s> using cache", url.GetFileName().c_str());
 
-  m_sourcePath = url.Get();
```

Comment:
Would you accept the tidy-up patches in a separate pull-request? I've encountered a fair amount of redundant code hanging around.